AcloConcurrency/AcloConcurrency/Collections/Views/SearchView.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  SearchView.swift
+//  AlcoConcurrency
+//
+//  Created by DariaMikots on 14.06.2022.
+//
+
```

Comment:
Delete this pls and in other places also

Reply:
fix